test/support/utils.rb

```diff
@@ -11,7 +11,22 @@ def type(command)
     end
 
     def create_message fail_msg
-      "#{fail_msg} on #{@mode} mode\n[DEBUG SESSION LOG]\n> " + @backlog.join('> ')
+      "#{fail_msg} on #{@mode} mode\n[DEBUGGER SESSION LOG]\n> #{@backlog.join('> ')}#{debuggee_backlog}"
+    end
+
+    def debuggee_backlog
+      return if @mode == 'LOCAL'
+
+      backlog = []
+      begin
+        Timeout.timeout(TIMEOUT_SEC) do
+          while (line = @remote_r.gets)
```

Comment:
it can raise EIO error, so you need to rescue it.

Reply:
Thanks!
I'll change it.

Reply:
https://github.com/ruby/debug/pull/132/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R27